packages/commonwealth/server/migrations/20240724130410-migrate-community-category-data-to-community-tags-table.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.sequelize.transaction(async (t) => {
```

Comment:
can you try to write a single update query here? looping through all communities with promises can be inefficient if the set is large

Reply:
Thank @Rotorsoft, updated the migration. Plz re-approve PR if the changes work.

Reply:
you can probably optimize this a little bit more by using a CTE to find all communities with those two tags first and avoid the subqueries, but probably not a big deal given the number of communities we have in prod... approving 👍